src/v0/destinations/am/networkHandler.js

```diff
@@ -0,0 +1,61 @@
+const { ThrottledError, AbortedError, RetryableError } = require('@rudderstack/integrations-lib');
+const { processAxiosResponse } = require('../../../adapters/utils/networkUtils');
+const { prepareProxyRequest, proxyRequest } = require('../../../adapters/network');
+const { isHttpStatusSuccess, isHttpStatusRetryable } = require('../../util');
+const logger = require('../../../logger');
+const { DESTINATION } = require('./config');
+
+const responseHandler = (responseParams) => {
+  const { destinationResponse } = responseParams;
+  const message = `[${DESTINATION} Response Handler] - Request Processed Successfully`;
+  const { status, response } = destinationResponse;
+  if (isHttpStatusSuccess(status)) {
+    const { error, code } = response;
+    if (code === 429) {
+      // log the error when it contains 'Too many requests for some devices and users' and throw a RetryableError
+      if (error.includes('Too many requests for some devices and users')) {
+        logger.error('Too many requests for some devices and users.');
```

Comment:
can you put destination id and destination type here

Reply:
destination type (amplitude is already there), destinationId should be coming in statTags not sure how to get here in the payload though